sdk/eventhub/event-hubs/test/perf/track-1/README.md

```diff
@@ -0,0 +1,14 @@
+### Guide
+
+1. Navigate to `test-utils\perfstress` folder `cd sdk\test-utils\perfstress\`
+2. Build the package `rush update && rush build -t test-utils-perfstress`
+3. Pack the perf package `rushx pack`
+4. Navigate to `event-hubs\perf\track-1` folder `cd sdk\eventhub\event-hubs\perf\track-1`.
+5. Install the perf package `npm i ..\..\..\..\..\test-utils\perfstress\azure-test-utils-perfstress-1.0.0.tgz`
```

Comment:
Should we just include this in the package.json of the perf test? Then we can at least eliminate this step.

Reply:
Hmm, I added it in storage's package.json, can do the same here.
You raise a good point that we eliminate this step.

Reply:
It doesn't look like the package.json for the perf test was updated to reference the perf package.

Reply:
Oops, added now!